### DISCUSSION TITLE
Academic conference 2023 추천 책

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#197] [General]
둘다 좋은 책입니다. 

내년에 선정할 책으로 참고해 보겠습니다.

요렇게만 써 놓으면 또 잊을 수 있으니까 issue로 뽑고 discussion으로 옮겨 놓겠습니다.

_Originally posted by @jongfeel in https://github.com/ThinkAboutSoftware/AcademicConference/pull/194#discussion_r846131219_

**Comment by Joseph-Cha** [comment]
코멘트로 남긴 책은 아래 2가지 책입니다.

- [CODE 코드 하드웨어와 소프트웨어에 숨어 있는 언어 (찰스 펫졸드 저)](http://www.yes24.com/Product/Goods/16667186)
- [피플웨어 (톰 드마르코 저)](http://www.yes24.com/Product/Goods/13657193)

**Comment by wooyaggo86** [comment]
- Software Engineering at Google

**Reply by wooyaggo86** [reply]
[개발 7년차, 매니저 1일차](http://www.kyobobook.co.kr/product/detailViewKor.laf?ejkGb=KOR&mallGb=KOR&barcode=9791162242551&orderClick=LAG&Kc=)

**Reply by wooyaggo86** [reply]
[실리콘밸리의 팀장들](http://www.kyobobook.co.kr/product/detailViewKor.laf?ejkGb=KOR&mallGb=KOR&barcode=9788935212828&orderClick=LEA&Kc=)

**Reply by wooyaggo86** [reply]
프로그래머의 뇌

**Reply by wooyaggo86** [reply]
좋은 코드, 나쁜 코드

**Reply by wooyaggo86** [reply]
리팩터링

**Reply by wooyaggo86** [reply]
소프트웨어 아키텍처 101 엔지니어링 접근 방식으로 배우는 소프트웨어 아키텍처 기초
http://www.kyobobook.co.kr/product/detailViewKor.laf?mallGb=KOR&ejkGb=KOR&barcode=9791162244869&orderClick=JBD

**Reply by wooyaggo86** [reply]
Software Architecture: The Hard Parts
https://www.amazon.sg/Software-Architecture-Trade-Off-Distributed-Architectures/dp/1492086894/ref=asc_df_1492086894/?tag=googleshoppin-22&linkCode=df0&hvadid=405606626615&hvpos=&hvnetw=g&hvrand=10641985459964247422&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9062530&hvtargid=pla-1294786739508&psc=1

번역서가 없는 것 같네요.. ㅜ

**Comment by TaeHyoungKwon** [comment]
1. [도메인 주도 설계](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=12174216) - 에릭에반스
3. [대체 뭐가 문제야?](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=22740732) - 제럴드 와인버그
    - [이 영상](https://youtu.be/3H4umWD5bwI?t=15) 에서 말하는 엘리베이터 문제가 이 책에 나오는 내용입니다
4. [테크니컬 리더](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=22740964) - 제럴드 와인버그
5. [프로그래밍 심리학](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=34228223) - 제럴드 와인버그
6. [린 애자일 기법을 활용한 테스트 주도 개발](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=40990966)

**Comment by jongfeel** [comment]
으악 다 좋은 책들로 얘기해 주셔서 감사합니다.

하지만 내년에 논의할 책은 세 권만 정해야 해서
고르기가 너무 어렵군요

**Reply by Joseph-Cha** [reply]
제비뽑기를 한번 해보는 건 어떨까요?ㅎㅎ

**Reply by wooyaggo86** [reply]
책을 카테고리별로 나눠서 선정하는 건 어떨까요?
일단 임의로 생각해본걸 적어봅니다.
- 기본서
- 아키텍처
- 팀빌딩
- 애자일
- 테스트

**Comment by jongfeel** [comment]
실리콘밸리 팀장들(종필)
좋은 코드, 나쁜 코드(동훈)
소프트웨어 아키텍처 101(태형)

우연치 않게 우현님이 추천해 주신 책으로 다들 선정했습니다.
그런데 이렇게 정해 놓고 우현님 안오시면??? ㅋㅋㅋ

**Reply by wooyaggo86** [reply]
앗?!🥲
